rpm/xcache.spec

```diff
@@ -202,6 +203,9 @@ mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/95-cms-xcache-logging.cfg
 
 %changelog
+* Mon Aug 5 2109 Edgar Fajardo <emfajard@ucsd.edu> - 1.1.1-0.1
```

Comment:
Could you update the changelog to reflect 1) the 2nd RC and 2) that this is the 3rd RC?